From 7198387767e36af89c89f60a2060a11d4959ff53 Mon Sep 17 00:00:00 2001
From: Junping Du <junping_du@apache.org>
Date: Sat, 9 May 2015 06:11:13 -0700
Subject: [PATCH 1759/2848] MAPREDUCE-6359. In RM HA setup, Cluster tab links
 populated with AM hostname instead of RM.
 Contributed by zhaoyunjiong.

(cherry picked from commit df36ad0a08261b03c250b6f745b27e5f83e4286e)

Change-Id: I54d2e23281e82a3d0abb59035c166548d95ce408
---
 .../mapreduce/v2/app/webapp/AppController.java     |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/webapp/AppController.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/webapp/AppController.java
index d7929cc..108ba28 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/webapp/AppController.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/webapp/AppController.java
@@ -63,7 +63,7 @@ protected AppController(App app, Configuration conf, RequestContext ctx,
     set(APP_ID, app.context.getApplicationID().toString());
     set(RM_WEB,
         JOINER.join(MRWebAppUtil.getYARNWebappScheme(),
-            WebAppUtils.getResolvedRMWebAppURLWithoutScheme(conf,
+            WebAppUtils.getResolvedRemoteRMWebAppURLWithoutScheme(conf,
                 MRWebAppUtil.getYARNHttpPolicy())));
   }
 
-- 
1.7.9.5

